### DISCUSSION TITLE
Can you pass a WebSocket reference to initialize an existing WebSocket?

### DISCUSSION THREAD

**Discussion by joedborg** [encode/starlette#1948] [Q&A] [answered]
Kind of, but not directly related to https://github.com/encode/starlette/issues/216.

I have an example where I need to pass work off to a subprocess, for security, and would like to be able to pick the WebSocket connection off to that, rather than reconcile the output in the main web server.

Is it possible to initalize the same WebSocket connection within the subprocess?

**Comment by Kludex** [comment]
If it's possible, I don't know how to do it. :eyes:

**Comment by adriangb** [comment, marked as answer]
Generally you can't share network connections with subprocesses
